conf/vertebrates/mlss_conf.xml

```diff
@@ -34,69 +34,69 @@
-    <one_vs_all method="LASTZ_NET" ref_genome="canis_lupus_familiaris" against="Carnivora" ref_amongst="Eutheria"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="felis_catus" against="Feliformia" ref_amongst="Carnivora"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="equus_caballus" against="Perissodactyla"/>
 
-    <one_vs_all method="LASTZ_NET" ref_genome="bos_taurus" ref_amongst="Eutheria">
+    <one_vs_all method="LASTZ_NET" ref_genome="bos_taurus">
       <species_set in_collection="default">
-        <taxonomic_group taxon_name="Cetartiodactyla"/>
+        <taxonomic_group taxon_name="Artiodactyla"/>
       </species_set>
     </one_vs_all>
 
-    <one_vs_all method="LASTZ_NET" ref_genome="ovis_aries" against="Caprinae" ref_amongst="Cetartiodactyla"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="sus_scrofa" against="Sus"/>
 
-    <!-- Early vertebrates and other chordates -->
-    <one_vs_all method="LASTZ_NET" ref_genome="ciona_intestinalis" against="Ciona" ref_amongst="Cyclostomata"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="petromyzon_marinus" against="Cyclostomata" ref_amongst="Ciona"/>
+    <!-- Sauropsids -->
+    <one_vs_all method="LASTZ_NET" ref_genome="gallus_gallus">
+      <species_set>
+        <taxonomic_group taxon_name="Sauropsida"/>
+        <!-- Remove all the taxa that have a better reference than chicken -->
+        <taxonomic_group taxon_name="Iguania" exclude="1"/>
+      </species_set>
+    </one_vs_all>
 
-    <!-- These genomes are currently the only ones in their clades, so no alignment within the "against"/"species_set" -->
-    <!-- However they will be aligned against the other reference genomes in "ref_amongst" -->
-    <one_vs_all method="LASTZ_NET" ref_genome="equus_caballus" against="Perissodactyla" ref_amongst="Laurasiatheria"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="sus_scrofa" against="Suina" ref_amongst="Laurasiatheria"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="anolis_carolinensis" against="Lepidosauria" ref_amongst="Sauropsida"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="pelodiscus_sinensis" against="Testudines" ref_amongst="Sauropsida"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="xenopus_tropicalis" against="Amphibia" ref_amongst="Euteleostomi"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="ornithorhynchus_anatinus" against="Prototheria" ref_amongst="Euteleostomi"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="latimeria_chalumnae" ref_amongst="Euteleostomi">
+    <one_vs_all method="LASTZ_NET" ref_genome="anolis_carolinensis" against="Iguania"/>
+
+    <!-- Fishes -->
+    <one_vs_all method="LASTZ_NET" ref_genome="oryzias_latipes">
       <species_set>
-        <!--Coelacanths and lungfishes -->
-        <taxonomic_group taxon_name="Sarcopterygii"/>
+        <!-- Include urochordates, cephalochordates, cyclostomes, cartilaginous fishes, ray-finned fishes, coelacanths and lungfishes -->
```

Comment:
This will also include amphibians, do we need the only amphibian we have to be aligned to both human and a fish?

Reply:
This is certainly the **most** questionable species set.